README.rst

```diff
@@ -1,25 +1,47 @@
+.. raw:: html
+
+   <p align="center">
+     <img width="450" height="200" src=docs/tialab_logo.png>
```

Comment:
I noticed that this relative import breaks when sphinx imports the readme. Can use a full GitHub path or path to the image hosted elsewhere instead.

Reply:
e.g. so it will load on readthedocs

Reply:
Add link to https://warwick.ac.uk/fac/sci/dcs/research/tia/models/tialab_logo.png in this commit https://github.com/TIA-Lab/tiatoolbox/commit/e80e860f7b3a4e0fd1a934290556b6f301a8c41b.